tests/testPhotoCal.py

```diff
@@ -134,36 +133,57 @@ def testFlags(self):
             mapper.addMapping(self.srcCat.schema.find(name).key)
         cat.extend(self.srcCat, mapper=mapper)
 
-        #   test that by default, no stars are reserved and used < candidates
+        # test that by default, no stars are reserved and all used are candidates
         task.run(exposure=self.exposure, sourceCat=cat)
-        candidates = 0
         used = 0
-        reserved = 0
         for source in cat:
-            if source.get("calib_photometryCandidate"):
-                candidates += 1
             if source.get("calib_photometryUsed"):
+                self.assertTrue(source.get("calib_photometryCandidate"))
                 used += 1
-            if source.get("calib_photometryReserved"):
-                reserved += 1
-        self.assertLessEqual(used, candidates)
-        self.assertEqual(reserved, 0)
+            self.assertFalse(source.get("calib_photometryReserved"))
+        # test that some very actually used
```

Comment:
very => are

"some" what?  And you're really interested in the proper setting of flags (i.e. if an object was used in the fit, was `calib_photometryUsed` set to **True**)